laghos.cpp

```diff
@@ -757,17 +757,47 @@ int main(int argc, char *argv[])
         // -restore phase
         // No need to specify t_final because the loop in -restore phase is determined by the files in ROMsol folder.
         // When -romhr or --romhr are used in -online phase, then -restore phase needs to be called to project rom solution back to FOM size
+        std::ifstream infile_tw_steps("run/tw_steps");
+        int nb_step;
+        nb_step = 1000000;
+        cout << "nb_step: " << nb_step << "\n";
```

Comment:
Why print here? Isn't this set later, so that it should be printed later? The same goes for rom_dim* in the next 3 lines.

Reply:
These prints were there for my debugging purpose. I forgot to remove them. Now, I removed them. Thank you for pointing it out.